plugins/plugin-testing-java/plugin-testing-testng/che-plugin-testing-testng-server/src/main/java/org/eclipse/che/plugin/testing/testng/server/TestNGRunner.java

```diff
@@ -187,32 +175,7 @@ public String getName() {
   }
 
   @Override
-  protected boolean isTestMethod(IMethod method, ICompilationUnit compilationUnit) {
-    return javaTestFinder.isTest(
-        method, compilationUnit, JavaTestAnnotations.TESTNG_TEST.getName());
-  }
-
-  @Override
-  protected boolean isTestSuite(String filePath, IJavaProject project) {
-    int projectPathLength = project.getPath().toString().length();
-    String path = filePath.substring(projectPathLength, filePath.length());
-    IFile file = project.getProject().getFile(path);
-
-    if (!file.exists()) {
-      return false;
-    }
-
-    SAXParserFactory factory = SAXParserFactory.newInstance();
-    TestNGSuiteParser suiteParser = new TestNGSuiteParser();
-    try {
-      SAXParser parser = factory.newSAXParser();
-      parser.parse(file.getContents(), suiteParser);
-    } catch (ParserConfigurationException | SAXException | IOException e) {
-      LOG.error("It is not possible to parse file " + path, e);
-    } catch (CoreException e) {
-      LOG.error("It is not possible to read file " + path, e);
-    }
-
-    return suiteParser.isSuite();
+  protected String getTestAnnotation() {
+    return JavaTestAnnotations.TESTNG_TEST.getName();
```

Comment:
Do we even need a shared "JavaTestAnnotation" class? The values are never used in a shared context.

Reply:
JavaTestAnnotation is redundant class, i've just removed it